packages/components/src/components/tab-header/tab-header.tsx

```diff
@@ -33,10 +33,8 @@ export class TabHeader {
   /** True for a disabled Tabnavigation */
   @Prop() disabled?: boolean = false;
   /** True for smaller height and font size */
-  // DEPRECATED - size should replace small
+  /** @deprecated - css overwrites should replace small */
   @Prop() small?: boolean = false;
-  /** (optional) size  */
-  @Prop() size: 'small' | 'large' = 'large';
```

Comment:
This we want to keep (so it's not a "breaking" change) 🙂

We add the `statusNote` the same way as with the `small` prop (and the "DEPRECATED" comment).

So if somebody uses either `size` or `small` they will get the warning in the console. But the prop has actually no effect on the component.

Reply:
Had already wondered extremely about this. Thanks for the clarification 🙂